### ISSUE TITLE
Surf_Island Elevator Shaft bug

### ISSUE THREAD

**Issue by Wh1t3Rose** [fluffyst/Surftimer#113]
Hello I have a bug on Surf_Island where after you get past the lasers and you float up the elevator shaft you just keep going up through the roof and you never descend, Then it resets you back to the start zone. I have no idea how to go about troubleshooting this issue, 

I've tried different versions of the map and I have tried using !mg to adjust gravity fix to no avail.

**Comment by dPexxIAM** [comment]
!ms > disable gravity fix

**Comment by Wh1t3Rose** [comment]
> 
> 
> !ms > disable gravity fix

Read the last sentence of my comment dude....

**Comment by dPexxIAM** [comment]
xD it was late ok...

Map has always worked fine for me, perhaps it's some other plugin you're running that's messing with it.

**Comment by Wh1t3Rose** [comment]
> 
> 
> xD it was late ok...
> 
> Map has always worked fine for me, perhaps it's some other plugin you're running that's messing with it.

Yea maybe. I need to setup a Dev server and test it with nothing but the Surf Plugin and surf_island and see if the bug is reproducible. I'll do that this weekend perhaps....

**Comment by fluffyst** [comment]
the gravity fix toggle fixes this fine on my server, any more info?

**Comment by kmsraidboss** [comment]
Gravity fix toggles do not appear to work on any of my servers either. All running the latest version here, as well as SM 1.10. There are no Sourcemod errors or logs showing anything. The regular console log shows nothing. I've tried enabling/disabling the gravity fix in the CFG, via !ms menu on each problem map, and even directly toggling it in the database. No matter what, any map with low gravity sends players to the ceiling and the maps are essentially unplayable. 

Just wanted to give an additional comment here about this because it's still an on-going issue as far as I know and a relatively game breaking one for many maps.

**Comment by DigidyDog** [comment]
I "fixed" this by using stripper to modify the gravity trigger making it a trigger push with a speed of 1700. The map might not be the exact same as the original but its doable now. It also mimics it pretty well.

**Comment by flaggx** [comment]
> I "fixed" this by using stripper to modify the gravity trigger making it a trigger push with a speed of 1700. The map might not be the exact same as the original but its doable now. It also mimics it pretty well. It also doesn't require editing the map in hammer or making users download a new map file.

Do you have the stripper filter for this map? I would like to use it.

**Comment by DigidyDog** [comment]
Yeah, after I did this I the disabling the gravity fix is what actually fixed it for me but I will upload the file.

**Comment by flaggx** [comment]
> @flaggx
> 
> > Do you have the stripper filter for this map? I would like to use it.
> 
> Yeah, after I did this I found disabling the gravity fix is what actually fixed it for me so I removed this file.
> 
> ```
> modify:
> {
> 	match:
> 	{
> 	"model" "*114"
> 	}
> 	replace:
> 	{
> 	"classname" "trigger_push"
> 	}
> 	delete:
> 	{
> 	"gravity" "-.5"
> 	}
> 	insert:
> 	{
> 	"pushdir" "-90 0 0"
> 	"speed" "1700"
> 	}
> }
> ```
> 
> If for some reason you have a different version of the map or something, you might want to change "gravity" "-.5" to "gravity" "/.*/"
> 
> * oh and btw having the gravity fix enabled globally and disabling it per map is how I have my configured and for some reason disabling the fix per map is actually what fixes it for me.

Very cool thank you! I will try disabling again first. If not I will use this.
